docs/src/components/landing/Banner.js

```diff
@@ -10,54 +11,67 @@ import KeyboardArrowRightRounded from '@mui/icons-material/KeyboardArrowRightRou
 export default function Banner(props) {
   const { content } = props;
   return (
-    <Container sx={{ py: { xs: 2, sm: 4, md: 8 } }}>
-      <Stack
-        sx={{
-          borderRadius: 1,
-          p: 2,
-          bgcolor: (theme) => (theme.palette.mode === 'dark' ? 'primaryDark.900' : 'primary.50'),
-          border: '1px solid',
-          borderColor: (theme) =>
-            theme.palette.mode === 'dark' ? 'primaryDark.500' : 'primary.100',
-          display: 'flex',
-          flexDirection: {
-            xs: 'column',
-            sm: 'row',
-          },
-          justifyContent: 'space-between',
-          alignItems: { xs: 'flex-start', sm: 'center' },
-        }}
-      >
-        <div>
-          <Typography fontWeight="bold" sx={{ mb: 0.5 }}>
-            {content.title}
-          </Typography>
-          <Typography variant="body2" color="text.secondary" sx={{ maxWidth: 700 }}>
-            {content.description}
-          </Typography>
-        </div>
-        <Button
-          component={Link}
-          noLinkStyle
-          data-ga-event-category="ToolpadLanding"
-          data-ga-event-label={content.action.label}
-          data-ga-event-action="Upvote"
-          target="_blank"
-          rel="noopener"
-          href={content.action.href}
-          variant="contained"
-          fullWidth
-          endIcon={<KeyboardArrowRightRounded />}
+    <Container
+      sx={{
+        pb: { xs: 2, sm: 4, md: 8 },
+        pt: { xs: 0, sm: 1, md: 2 },
+        bgcolor: (theme) =>
+          theme.palette.mode === 'dark' ? theme.palette.primaryDark[900] : theme.palette.grey[50],
+      }}
+      maxWidth="100vw"
```

Comment:
is it possible to use something more relative to the parent elements instead of viewport units?